src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.Bytes.cs

```diff
@@ -39,11 +36,8 @@ private void WriteBase64StringHelper(ReadOnlySpan<byte> utf8PropertyName, ReadOn
         /// <summary>
         /// Writes the property name and raw bytes value (as a Base64 encoded JSON string) as part of a name/value pair of a JSON object.
         /// </summary>
-        /// <param name="propertyName">The property name of the JSON object to be transcoded and written as UTF-8.</param>
-        /// <param name="bytes">The binary data to be written as a base 64 encoded JSON string as part of the name/value pair.</param>
-        /// <remarks>
-        /// The property name is escaped before writing.
```

Comment:
@steveharter if I passed down a escaped string to this API, would it escape it again? Therefore does that mean we still need this remark for APIs that take string/ROS?

Reply:
Yes, it gets re-escaped.  If you pass in "\u0030" it will get extra-escaped to "\\u0030" so that after going onto and back off of the wire GetString() returns the original input.

Only JsonEncodedString doesn't re-escape (because it already represents the properly escaped form for a particular value).

Reply:
So therefore, that means that we indeed need this remark for these overrides that don't take a `JsonEncodedString`... thanks for confirming.